core/src/main/java/nl/nn/adapterframework/filesystem/FileSystemActor.java

```diff
@@ -558,16 +593,19 @@ public MessageOutputStream provideOutputStream(PipeLineSession session, IForward
 
 
 
-	protected void addActions(List<String> specificActions) {
+	protected void addActions(List<FileSystemAction> specificActions) {
 		actions.addAll(specificActions);
 	}
 
-	@IbisDoc({"1", "Possible values: "+ACTION_LIST+", "+ACTION_INFO+", "+ACTION_READ1+", "+ACTION_READ_DELETE+", "+ACTION_MOVE+", "+ACTION_COPY+", "+ACTION_DELETE+", "+ACTION_MKDIR+", "+ACTION_RMDIR+", "+ACTION_WRITE1+", "+ACTION_APPEND+", "+ACTION_RENAME+". If parameter ["+PARAMETER_ACTION+"] is set, then the attribute action value will be overridden with the value of the parameter.", "" })
+	@IbisDoc({"1", "If parameter ["+PARAMETER_ACTION+"] is set, then the attribute action value will be overridden with the value of the parameter.", "" })
 	public void setAction(String action) {
-		this.action = action;
+		this.action = EnumUtils.parse(FileSystemAction.class, action);
+	}
+	public FileSystemAction getActionEnum() {
+		return action;
 	}
```

Comment:
Nowadays, you can use `public void setAction(FileSystemAction action)` and `public FileSystemAction getAction()`. The digester interprets the Enum, and the FrankDoc understands it too.